app/components/project-geometries/modes/draw/feature-label-form.js

```diff
@@ -18,13 +18,50 @@ export default class FeatureLabelFormComponent extends Component {
   @argument
   options=null;
 
+  isBold = false;
+
+  isLarge = false;
+
+  @observes('selectedFeature')
+  selectedFeatureUpdated() {
+    this.set('isBold', false);
+    this.set('isLarge', false);
+  }
+
   @alias('selectedFeature.features.firstObject.properties.label')
   label;
 
+  @alias('selectedFeature.features.firstObject.geometry.type')
+  geomType;
+
   @action
   handleSelectChange(newLabel) {
     this.set('label', newLabel);
-    this.updateSelectedFeature(newLabel);
+    this.updateSelectedFeature('label', newLabel);
+    if (this.get('geomType') === 'Polygon') {
+      this.updateSelectedFeature('textFont', 'bold');
+    }
```

Comment:
This is starting to reveal the need for subclassing of some sort (https://sourcemaking.com/refactoring/replace-type-code-with-subclasses). We have all this geometry type-specific handled for a feature-label-form. Maybe geometry-types are _labelable_ things and get access to some service for that? Anyways, for another time...